sabnzbd/downloader.py

```diff
@@ -853,13 +853,19 @@ def __finish_connect_nw(self, nw: NewsWrapper) -> bool:
                     self.plan_server(server, _PENALTY_TOOMANY)
                     server.threads -= 1
             elif error.code in (502, 481, 482) and clues_too_many_ip(error.msg):
-                # Account sharing?
+                # Login from (too many) different IP addresses
                 if server.active:
-                    errormsg = T("Probable account sharing") + display_msg
+                    errormsg = T("Login from (too many) different IP addresses") + display_msg
```

Comment:
Agree partially with @puzzledsab, let's make it `Login from too many different IP addresses to server %s`.